src/Microsoft.SqlTools.ServiceLayer/Connection/ConnectionService.cs

```diff
@@ -1328,8 +1328,10 @@ public static SqlConnectionStringBuilder CreateConnectionStringBuilder(Connectio
                             connStringBuilder.Password = ConnectionService.PasswordPlaceholder;
                         }
 
-                        connStringBuilder.ApplicationName = "sqlops-connection-string";
-
+                        if (connStringParams.IncludeApplicationName)

```

Comment:
This should be defaulted to true in STS so we don't have to update every client that uses it

Reply:
This is defaulted to true in mssql-vscode. This won't update every client since it already defaulted and only would change when someone sets this parameter to false.

Reply:
More than just VS Code uses STS 😄 

The default here should be that if it isn't specified the behavior stays the same as previously.

Reply:
Makes sense to set this to true then.